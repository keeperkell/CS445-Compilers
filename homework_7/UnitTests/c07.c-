## C-S22
main()
 begin 
    bool x[3];
    bool y[4];

    x[2] <- true;
    y[1] <- false;
    outputb(x[2]  or  y[1]);
    outnl();
 end 
