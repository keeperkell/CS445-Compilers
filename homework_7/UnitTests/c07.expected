Number of warnings: 0
Number of errors: 0
------------------------------------
Loading file: UnitTests/c07.tm
T
Bye.
